src/gordon_gcp/plugins/service/event_consumer.py

```diff
@@ -264,6 +274,7 @@ def __init__(self, config, success_channel, error_channel, metrics,
         self._message_schemas = validator.schemas.keys()
         self._loop = loop
         self._logger = logging.getLogger('')
+        self._max_msg_age = config['max_msg_age']
```

Comment:
It might be worth adding `seconds` or even just `_s` in the var names somewhere. It's easy to forget what units things are in.

Reply:
Are you including the name in the config file in that?

Reply:
There are other config vars that are seconds but aren't obvious.  We agreed to hold off because they should be consistent, but we shouldn't be doing all of them now.  Also, '300' as the example value should be fairly obvious, one hopes.